data-esnext.js

```diff
@@ -91,6 +91,14 @@ exports.tests = [
       name: 'basic support',
       mdn: 'https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/SIMD',
       exec: function () {/*
+        simdFloatTypes=['Float32x4'];
```

Comment:
hmm - due to the eval-like nature of the tests, i don't think this will actually work. I think they might need to be globals, or else inlined into each test :-/